sdk/runanywhere-commons/scripts/build-ios.sh

```
@@ -222,6 +229,61 @@ build_macos() {
+        MLX_FLAGS="-DMLX_INCLUDE_DIR=${MLX_INC}"
+        log_info "MLX include dir: ${MLX_INC}"
+    else
+        log_warn "MLX not found. Steel attention/gemm kernels may fail to compile."
+    fi
+
+    cmake "${METALRT_ROOT}" \
+        -DCMAKE_TOOLCHAIN_FILE="${PROJECT_ROOT}/cmake/ios.toolchain.cmake" \
+        -DIOS_PLATFORM="OS" \
+        -DIOS_DEPLOYMENT_TARGET="${IOS_DEPLOYMENT_TARGET}" \
+        -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" \
+        $MLX_FLAGS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)